lib/core/test/unit/Cardano/Wallet/Primitive/Types/TokenMapSpec.hs

```diff
@@ -1051,7 +1050,7 @@ instance Arbitrary TokenQuantity where
     -- verify that the token map invariant (that a map contains no
     -- zero-valued tokens) is maintained.
     arbitrary = genTokenQuantitySmall
-    shrink = shrinkTokenQuantitySmall
+    shrink = shrinkTokenQuantitySmallPositive
```

Comment:
Hi @rvl 

I think this still makes a change to the behaviour. If possible, can this also be moved to the other PR?

Reply:
Original shrinking behaviour restored in 83e51d530bc3118ad6828dee9caf0cef6bcedd51.

Reply:
OK thanks. :+1: